src/components/Messages/MessageInput.tsx

```
@@ -17,6 +22,46 @@ export default function MessageInput({
+      toast.info("Scanning audio for voice cloning/deepfake anomalies...");
+
+      try {
+        const { valid, error } = await DeepfakeAudioDetectorService.validateAudioFile(file);
+
+        if (!valid) {
+          toast.error(error || "Upload blocked: Deepfake audio detected (Impersonation/Generative AI Fraud).");
+        } else {
+          toast.success("Audio verified: No synthetic voice artifacts detected.");
+          // Append audio filename to input message
+          setInputMessage(
+            inputMessage ? `${inputMessage} [Audio Attachment: ${file.name}]` : `[Audio Attachment: ${file.name}]`
```

ESLint

[error] Replace `·?·`${inputMessage}·[Audio·Attachment:·${file.name}]`·:·`[Audio·Attachment:·${file.name}]`` with `⏎··············?·`${inputMessage}·[Audio·Attachment:·${file.name}]`⏎··············:·`[Audio·Attachment:·${file.name}]`,`

(prettier/prettier)